src/main.c

```diff
@@ -84,9 +141,26 @@ int main(int argc, char *argv[])
     uint32_t loop_cnt = 0;
     int timer_sched_loop_interval;
 
+    /**
+     * add application agruments parse before EAL ones.
+     * use it like the following:
+     * ./dpvs -v
+     * OR
+     * ./dpvs -- -n 4 -l 0-11 (if you want to use eal arguments)
+     */
+    err = parse_app_args(argc, argv);
+    if (err < 0) {
+        fprintf(stderr, "fail to parse application options\n");
+        exit(EXIT_FAILURE);
+    }
+    argc -= err, argv += err;
+
     /* check if dpvs is running and remove zombie pidfile */
     if (dpvs_running(DPVS_PIDFILE)) {
-        fprintf(stderr, "dpvs is already running\n");
+        fprintf(stderr, 
+            "dpvs is already running\ndpvs version: %s, build on %s\n", 
+            DPVS_VERSION, 
```

Comment:
The question I've raised still not get resolved, if print version here, user may get wrong version for old dpvs which is already running. Should not print version here I think.